docs/ScoreCard.md

```diff
@@ -0,0 +1,55 @@
+# ScoreCard
+Card component that calls attention to particular values.
+
+<div style="align-items: center; text-align: center; width: 100%; display: flex; justify-content: space-evenly; margin-bottom: 20px">
+    <img width="40%" alt="Score Card with multiple highlighted values" src="./images/scoreCard.png"><br/>
+    <img width="40%" alt="Score Card with single badge-style value" src="./images/scoreCard_alt.png"><br/>
+</div>
+
+## Usage
+
+<div style="align-items: center; text-align: center; width: 100%; display: flex; justify-content: space-evenly; margin-bottom: 20px">
+    <img width="80%" alt="Score Card with labels on different parts" src="./images/scoreCardAnatomy.png"><br/>
```

Comment:
you'll need a new image for anatomy since the API/usage differs from react

Reply:
let me know when the API is properly tuned and I can push new pics to this branch

Reply:
thanks @huayunh